<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>540</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to cart</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>520</y>
      <width>371</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="totalLabel">
       <property name="text">
        <string>Total: $0.00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove from cart</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="discountComboBox">
       <property name="currentText">
        <string>No discount</string>
       </property>
       <item>
        <property name="text">
         <string>No discount</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%10 discount</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="giftWrapping">
       <property name="text">
        <string>Gift Wrapping</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="applyDiscountButton">
       <property name="text">
        <string>Apply Discount</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>951</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QListWidget" name="productListWidget">
       <property name="modelColumn">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QListWidget" name="cartListWidget"/>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="productDetailsTextEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="confirmOrder">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>610</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Order</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cartNotification">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="discountNotification">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>610</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
